test/DrawBBox.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python2
+import subprocess 
+import numpy
+import sys
+
+# Edit this as required (windows, mac).  Be able to write this.
+PLY_FILE="/tmp/displaz_bbox.ply"
+
+# write .ply file representing a box with specfied center and transform to f
+def MakeBBox(v_center, transform, f):
```

Comment:
Did you see the python bindings inside the `bindings/python` directory?  They're pretty basic, but some ply writing utilities like this one could arguably go in there rather than in this example.  Just a thought really, I don't mind having this here, for the moment.